api/Element.json

```diff
@@ -5431,22 +5431,22 @@
               "version_added": null
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "≤14"
             },
             "safari": {
-              "version_added": null
+              "version_added": "3"
```

Comment:
Is this based on Chrome 1 timeline, or testing in Safari?

Reply:
This was based upon testing in Safari!

Reply:
Great, thanks!